src/components/DappsGallery/DappsGallery.module.scss

```diff
@@ -0,0 +1,28 @@
+.dappCard {
+  width: auto;
+  height: 100%;
+  box-sizing: border-box;
+  filter: drop-shadow(0px 5px 15px rgba(0, 0, 0, 0.12));
+  overflow: hidden;
+  border-radius: 20px;
+  -webkit-border-radius: 20px;
+  border: 1px solid rgba(255, 255, 255, 0.05);
+}
+
+.dappCardImage {
+  transition: all 300ms ease-out;
+  height: 100%;
+  width: 100%;
+  object-fit: fill;
```

Comment:
I suggest to replace with `contain` to not resize the image in a weird way like this:

<img width="664" alt="image" src="https://github.com/lukso-network/docs/assets/477945/0042ddea-08d6-4e79-96ea-cfea9f074549">


And to apply a background color to the boxes to fill the blank sapce